tools/peptideshaker/searchgui.xml

```diff
@@ -171,7 +171,7 @@
         <!-- Search Engine Selection -->
         <section name="search_engines_options" expanded="true" title="Search Engine Options">
             <param name="engines" type="select" display="checkboxes" multiple="True" label="DB-Search Engines">
-                <help>Comet and Tide shouldn't both be selected since they use a similar algoritm.</help>
+                <help>Comet and Tide shouldn't both be selected since they use a similar algoritm. OMSSA may not work into isolated environments like containers.</help>
                 <option value="X!Tandem" selected="True">X!Tandem</option>
                 <option value="MSGF" selected="True">MS-GF+</option>
                 <option value="OMSSA" selected="True">OMSSA</option>
```

Comment:
Should we disable OMSSA by default?

Reply:
Yes, you are probably right. I will disable it by default.